cts/settings/base.py

```diff
@@ -286,6 +286,8 @@
     },
 }
 CELERY_RESULT_BACKEND = None  # We never care about task results
+# Each instance needs its own queue
+CELERY_DEFAULT_QUEUE = 'queue_%s' % os.environ['INSTANCE']
```

Comment:
`INSTANCE` is set up on line 9, so we don't need to pull it from the environment again.